rspack.config.js

```diff
@@ -286,6 +286,7 @@ const config = {
     new rspack.EnvironmentPlugin({
       WEBPACK_BUNDLE: "development",
       MB_LOG_ANALYTICS: "false",
+      MB_BUG_REPORTING_ENABLED: process.env.MB_BUG_REPORTING_ENABLED || "false",
```

Comment:
instead of a build-time frontend environment variable, we should use a runtime backend environment variable